pyscriptjs/src/plugin.ts

```diff
@@ -87,15 +87,15 @@ export class PluginManager {
     }
 
     configure(config: AppConfig) {
-        for (const p of this._plugins) p.configure(config);
+        for (const p of this._plugins) p.configure?.(config);
 
         for (const p of this._pythonPlugins) p.configure?.(config);
     }
 
     beforeLaunch(config: AppConfig) {
         for (const p of this._plugins) {
             try {
-                p.beforeLaunch(config);
+                p?.beforeLaunch?.(config);
             } catch (e) {
                 logger.error(`Error while calling beforeLaunch hook of plugin ${p.constructor.name}`, e);
```

Comment:
I think we can remove the try/catch all over the lifecycle, this was my silly way to not have plugins blow up if they don't implement a certain method 😅 

I didn't know that you could do `?.` just before passing an argument to the function haha